tests/example-all-fail/.meta/config.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
this file is not needed, I guess

Reply:
Probably not, but it might in the future. I'd prefer to keep it.